docs/source/getting_started/dataset_creation/dataset_creation_local_server.rst

```diff
@@ -0,0 +1,57 @@
+.. _dataset-creation-local-server:
+
+Create a dataset with a local file server
+-----------------------------------------
+
+You can configure a dataset in the Lightly Platform that streams the images
+from your local file system. This makes the creation of a dataset much faster,
+as no upload of images and thumbnails is needd.
+Let's assume that your dataset is locally at `/projects/animals`.
+
+
+Setting up local file server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Navigate to the directory your dataset lies.
+From within this directory, create a local http server for the files:
+
+.. code-block:: bash
+
+    cd /projects/animals
+    http://localhost:1234/
```

Comment:
The command to start the server is missing no? At least on my system I have to do something like:

```
python -m http.server 1234
```